### PR TITLE
Update AboutUs page

### PR DESCRIPTION
**Pull request by rlrh**
Upload your image in `docs/images/{Github_username}.jpg`

### CODE REVIEW

**Comment by thomastanck** [comment]
@rlrh @epicfailname @qjqqyy Want to add your image into this PR?

**Comment by rlrh** [comment]
> @rlrh @epicfailname @qjqqyy Want to add your image into this PR?

Ok, I'll move my image to this PR

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -11,46 +11,40 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
...
-
-Role: Project Advisor
-
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
+=== Thomas Tan
+image::thomastanck.jpg[width="150", align="left"]
 {empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
```
Comment (qjqqyy) [inline docs/AboutUs.adoc]: we should be updating the GitHub URLs too right?

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -11,46 +11,40 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
```
Comment (qjqqyy) [inline docs/AboutUs.adoc:15]: We might want to remove the profile pictures of the se-edu team as they're no longer needed any more I believe.

**Review by qjqqyy — CHANGES_REQUESTED** [review]

File: docs/AboutUs.adoc (hunk 1 above)
Comment (rlrh) [reply docs/AboutUs.adoc]: Updated in 729a67d

File: docs/AboutUs.adoc (hunk 2 above)
Comment (rlrh) [reply docs/AboutUs.adoc:15]: Removed unused images in 729a67d

**Review by qjqqyy — APPROVED** [review]

File: docs/AboutUs.adoc [hunk 3]
```diff
@@ -5,52 +5,45 @@
...
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
-
-Role: Project Advisor
-
 '''
```
Comment (thomastanck) [inline docs/AboutUs.adoc:20]: I think this is a horizontal divider? It shouldn't be here probably.

File: docs/AboutUs.adoc [hunk 4]
```diff
@@ -5,52 +5,45 @@
...
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Thomas Tan
+image::thomastanck.png[width="150", align="left"]
+{empty}[http://github.com/thomastanck[github]]
```
Comment (thomastanck) [inline docs/AboutUs.adoc]: Maybe we should link to the https?

File: docs/AboutUs.adoc [hunk 5]
```diff
@@ -5,52 +5,45 @@
...
-Responsibilities: Dev Ops + Threading
+Role: Technical Lead, Developer +
+Responsibilities:
 
 '''
 
-=== Benson Meier
-image::yl_coder.jpg[width="150", align="left"]
-{empty}[http://github.com/yl-coder[github]] [<<johndoe#, portfolio>>]
+=== Rayner Lim
```
Comment (thomastanck) [inline docs/AboutUs.adoc]: Team Lead should be at the top.

**Review by thomastanck — CHANGES_REQUESTED** [review]

File: docs/AboutUs.adoc (hunk 4 above)
Comment (rlrh) [reply docs/AboutUs.adoc]: Fixed in d6d32ae

File: docs/AboutUs.adoc (hunk 3 above)
Comment (rlrh) [reply docs/AboutUs.adoc:20]: Fixed in d6d32ae

File: docs/AboutUs.adoc (hunk 5 above)
Comment (rlrh) [reply docs/AboutUs.adoc]: Fixed in d6d32ae

**Review by thomastanck — APPROVED** [review]

**Pull request merged**